src/m365/flow/commands/flow-get.ts

```diff
@@ -17,6 +17,16 @@ interface Options extends GlobalOptions {
   asAdmin: boolean;
 }
 
+interface Triggers {
```

Comment:
I suppose this interface describes a single trigger and should be named `Trigger`, right?